.travis.yml

```diff
@@ -20,6 +20,8 @@ matrix:
           env: DEPLOY=yes TASK_TESTS_COVERAGE=1
         - php: nightly
           env: TASK_SCA=1 COMPOSER_FLAGS="--ignore-platform-reqs" SYMFONY_DEPRECATIONS_HELPER=weak PHP_CS_FIXER_IGNORE_ENV=1
+        - php: 7.2
+          env: TASK_SCA=1 COMPOSER_FLAGS="--ignore-platform-reqs" SYMFONY_DEPRECATIONS_HELPER=weak PHP_CS_FIXER_IGNORE_ENV=1
```

Comment:
Tests pass so IMO we should also:
* update the `php` constraint in composer.json to allow PHP 7.2;
* update handling of `PHP_CS_FIXER_IGNORE_ENV` to always allow PHP 7.2;
* drop specific nightly flags here (`--ignore-platform-reqs`, `SYMFONY_DEPRECATIONS_HELPER` and `PHP_CS_FIXER_IGNORE_ENV`).

Also, test coverage is always done on the highest tested PHP version so it should be done on 7.2 now.

Reply:
I don't think we should move the coverage to a non-stable release of PHP.
Also, it seems that adding 7.2 to the allowed composer versions are blocked by #2907.

Reply:
No official support for 7.2 until we fix #2907 as @Jean85 claims :+1:
Fact that tests passed is caused by... not having tests for 7.2 syntax...

Reply:
What is there in #2907 that still prevents us for claiming PHP 7.2 support?

Reply:
please discuss it in proper thread
this PR claims to extend matrix to use both 7.2 and 7.3 for beta-tests, not about officially supporting 7.2